### ISSUE TITLE
Specify the requirements

### ISSUE THREAD

**Issue by gido** [frontend/toolbox-utils#28]
it looks like toolbox is not compatible with node 6 (use of trailing comma in function args list, use of async, etc).

It would be nice to see if we can be compatible with the maximum version of node and at least define with which version of nodejs this project is compatible.

**Comment by Yago** [comment]
Indeed, I saw that the last major version of our generator doesn't have the engine specified in the package.json.

I don't completely agree with the fact that we have to support a big range of Node's versions. Actually, the LTS (stable for prod) is in `8.10.0` and as you said, under `8.0.0`, there is no `async` support, it's a bit the Middle Ages of JavaScript and we must live in the present 😉 

We have to readd the engine definition in the `package.json` project with something like : 

```json
"engines": {
  "node": ">=8.0.0",
  "npm": ">=5.0.0"
},
```

**Comment by tonifisler** [comment]
This needs to be fixed, it's quite annoying. As @knuch said in #36, if you don't work with Node 12, it's broken. SMH

**Comment by Yago** [comment]
The weird fact is that I still haven't Node 12 installed on my Mac (even on the lat contrib) 😅

**Comment by Yago** [comment]
“it's broken” is not very relevant... What's broken exactly ?
